robottelo/utils/virtwho.py

```diff
@@ -535,3 +562,35 @@ def get_configure_command_option(deploy_type, args, org=DEFAULT_ORG):
     if deploy_type == 'name':
         return f"hammer -u {username} -p {password} virt-who-config deploy --name {args['name']} --organization '{org}' "
     return None
+
+
+def vw_fake_conf_create(
+    owner,
+    rhsm_hostname,
+    rhsm_username,
+    rhsm_encrypted_password,
+    fake_conf_file,
+    json_file,
+    is_hypervisor=True,
+):
+    conf_name = fake_conf_file.split("/")[-1].split(".")[0]
+    file = f'{fake_conf_file}\n'
+    title = f'[{conf_name}]\n'
```

Comment:
As you're already adding `\n` in cmd below, is `\n` again required here as well?

Reply:
@Gauravtalreja1 Thanks for your suggestion, have already updated

Reply:
@Gauravtalreja1 Could you please help to review the update? Thank you very much